libs/media-player/src/lib/MediaPlayer.tsx

```diff
@@ -1,12 +1,45 @@
-import styles from './MediaPlayer.module.css';
+import PauseIcon from '@mui/icons-material/Pause';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import { Button } from '@mui/material';
+import { MouseEventHandler, useEffect, useState } from 'react';
+import styles from './MediaPlayer.module.scss';
 
-/* eslint-disable-next-line */
-export interface MediaPlayerProps {}
+const useAudio = (audioUrl: string): [boolean, MouseEventHandler<HTMLButtonElement>] => {
+    const [audio] = useState(new Audio(audioUrl));
+    const [playing, setPlaying] = useState(false);
 
-export function MediaPlayer(props: MediaPlayerProps) {
+    const toggle: MouseEventHandler<HTMLButtonElement> = (): void => setPlaying(!playing);
+
+    useEffect(() => {
+        playing ? audio.play() : audio.pause();
+    }, [playing]);
+
+    useEffect(() => {
+        audio.addEventListener('ended', () => setPlaying(false));
+        return () => {
+            audio.removeEventListener('ended', () => setPlaying(false));
+        };
+    }, []);
+
+    return [playing, toggle];
+};
+
+export interface MediaPlayerProps {
+    audioUrl: string;
+}
+
+export function MediaPlayer({ audioUrl }: MediaPlayerProps) {
+    const [playing, toggle] = useAudio(audioUrl);
     return (
         <div className={styles['container']}>
-            <h1>Welcome to MediaPlayer!</h1>
+            <Button id={styles['radioButton']} variant="contained" onClick={toggle}>
+                Listen Live!
```

Comment:
this reminds me that you may want to experiment with the `Now Playing`

Reply:
Added fix to button, text will now change from "listen live" to "now playing".